<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePlayLauncher</class>
 <widget class="QWidget" name="GamePlayLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GamePlayLauncher</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTreeWidget" name="Friend_List">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>160</y>
     <width>271</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">2</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="GameStart">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>770</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="Friend_Request">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>120</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddFriend">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AddFriend</string>
   </property>
  </widget>
  <widget class="QLabel" name="User_Icon">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="User_ID">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="Main_Patch_Image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>871</width>
     <height>471</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Image</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="GameDesgin_Note">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Programming_Note">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Art_Note">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>560</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="PatNote_Link">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>851</width>
     <height>71</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;패치노트 링크&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>GameStart</zorder>
  <zorder>Friend_List</zorder>
  <zorder>Friend_Request</zorder>
  <zorder>AddFriend</zorder>
  <zorder>User_Icon</zorder>
  <zorder>User_ID</zorder>
  <zorder>Main_Patch_Image</zorder>
  <zorder>GameDesgin_Note</zorder>
  <zorder>Programming_Note</zorder>
  <zorder>Art_Note</zorder>
  <zorder>PatNote_Link</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resource/LauncherUI.qrc"/>
 </resources>
 <connections/>
</ui>
